lib/super_digit.rb

```diff
@@ -1,15 +1,25 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(log n) - where n is the number of digits
+# Space Complexity - O(log n) - where n is the number of digits
 def super_digit(n)
-   
+  if (super_helper(n) / 10) == 0
+    return super_helper(n)
```

Comment:
If you notice you're calling `super_helper(n)` several times, maybe store it in a variable to avoid recalculating.